### ISSUE TITLE
[BUG] Errors on limited videos (age, members, etc)

### ISSUE THREAD

**Issue by SpeckyYT** [SuspiciousLookingOwl/youtubei#13]
**Describe the bug**
Age-restricted and members-only videos throw errors on fetching.

**Versioning**
Module: `v0.0.1-rc.14`
Node: `v14.15.4`

**To Reproduce**
1) find an age-restricted/members-only video
2) `Client.getVideo('<the ID of the video>')`
3) observe result

**Expected behavior**
No errors (or another kind of "more-clear error", I guess)

**Real Behavior**
Age restricted:
```js
UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'find' of undefined
    at Video.load (C:\amogus\sus\mysusbot\node_modules\youtubei\dist\classes\BaseVideo.js:58:49)   
    at Video.load (C:\amogus\sus\mysusbot\node_modules\youtubei\dist\classes\Video.js:28:15)       
    at Client.<anonymous> (C:\amogus\sus\mysusbot\node_modules\youtubei\dist\classes\Client.js:68:50)
    at Generator.next (<anonymous>:null:null)
    at fulfilled (C:\amogus\sus\mysusbot\node_modules\youtubei\dist\classes\Client.js:5:58)        
    at runMicrotasks (<anonymous>:null:null)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
```
Members-only:
```js
UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'defaultText' of undefined
    at Video.load (C:\amogus\sus\mysusbot\node_modules\youtubei\dist\classes\BaseVideo.js:47:92)   
    at Video.load (C:\amogus\sus\mysusbot\node_modules\youtubei\dist\classes\Video.js:28:15)       
    at Client.<anonymous> (C:\amogus\sus\mysusbot\node_modules\youtubei\dist\classes\Client.js:68:50)
    at Generator.next (<anonymous>:null:null)
    at fulfilled (C:\amogus\sus\mysusbot\node_modules\youtubei\dist\classes\Client.js:5:58)        
    at runMicrotasks (<anonymous>:null:null)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
```

**Additional context**
For step 1 of the reproduction steps, here are two videos I tested it on:
```js
'https://www.youtube.com/watch?v=y7QkjrHmF18' // Age restricted
'https://www.youtube.com/watch?v=oPz6wk76buE' // Members-only
```

**Comment by SuspiciousLookingOwl** [comment]
Will be fixed on next patch, however age restricted / member-only videos will lose some functionality like accessing to `.upNext`, and `.related` will empty (so `.nextRelated()` will be useless). However I think there's a workaround for this, I will leave this issue open until I found it.

---

> or another kind of "more-clear error"

It's kind of difficult to do this since the data received from YouTube is a big pile of mess, different type of video (live video, normal video, member-only video, unlisted video, .etc) can have a different response schema, so finding the right path to the error message is difficult.

**Comment by SuspiciousLookingOwl** [comment]
Fixed in `0.0.1-rc.15`
